pyscriptjs/Makefile

```diff
@@ -0,0 +1,84 @@
+tag := latest
+git_hash ?= $(shell git log -1 --pretty=format:%h)
+
+base_dir ?= $(shell git rev-parse --show-toplevel)
+src_dir ?= $(base_dir)/src
+examples ?=  $(base_dir)/examples
+app_dir ?= $(shell git rev-parse --show-prefix)
+
+CONDA_ENV ?= ./env
+env := $(CONDA_ENV)
+
+
+.PHONY: clean
+clean:
+	find . -name \*.py[cod] -delete
+	rm -rf .pytest_cache .coverage coverage.xml
+
+clean-all: clean
+	rm -rf $(env) *.egg-info
+
+.PHONY: setup
+setup:
+	@if [ -z "$${CONDA_SHLVL:+x}" ]; then echo "Conda is not installed." && exit 1; fi
+	$(CONDA_EXE) env $(shell [ -d $(env) ] && echo update || echo create) -p $(env) --file environmentwh.yml
+
+.PHONY: shell
+shell:
+	@export CONDA_ENV_PROMPT='<{name}>'
+	@echo 'conda activate $(env)'
```

Comment:
I have been unable to coerce this into working in other repos. As-is, this is a no-op echo.

Reply:
Yeah, I was indirectly suggesting we remove it, unless you have a way to make it work :)